### ISSUE TITLE
Try new spatial matching methods

### ISSUE THREAD

**Issue by bcipolli** [bcipolli/lateralized-components#28]
From @atsuch:

>i also want to try a different way of RL matching. Rather than find a spatial match by directly comparing R and L components as currently implemented, do R- and L- matching through wb components

**Comment by atsuch** [comment]
I implemented it in the newRLmatch branch, but I found that in some cases signs of the maps are causing an issue. I think it would have been a problem regardless of the matching method, but basically, the scoring method doesn't care if the two maps have opposite signs, and it could match R and L maps with opposite signs. But that would create a concat RL image with flipped signs on R and L hemisphere, and such image becomes hard to match with any wb components. 

Example here.
![unmatched_rl_1](https://cloud.githubusercontent.com/assets/9255297/13783761/028faeae-eaa3-11e5-8179-14b101e3cf8a.png)
